ios/Inji.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -70,7 +70,7 @@
       "location" : "https://github.com/valpackett/SwiftCBOR",
       "state" : {
         "branch" : "master",
-        "revision" : "ec24382864e5ffc6d3915c0818745d5ab12545a8"
+        "revision" : "6b8232d6437e3fe7df8cc46e5bb872f7066e7d0d"
```

Comment:
why changed this?

Reply:
This is auto resolution of version. We have not specified any fixed commit.. so it resolved to the latest